saas_demo_test/README.rst

```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+ SaaS Demo Test
+================
+
+This addon is not meant to be used.
+
+Roadmap
+=======
+
+* After checking the fetch_and_generate_templates method, we should check whether the template is being created, but for this we need to pre-write the path to the test repository in the addons path, because we cannot reload the odoo in the tests.
```

Comment:
```suggestion
* Autotesting: We will need to check that template is created. Once it's done, be sure that there is a note in usage insturctions which says that the tests can be run with ......., because ..... 
```